packages/components/src/components/tag-dismissible/tag-dismissible-styles.ts

```diff
@@ -24,22 +25,23 @@ export const getComponentCss = (color: TagDismissibleColor, hasLabel: boolean):
         verticalAlign: 'top',
         outline: addImportantToRule(0),
       },
+      ...hostHiddenStyles,
       button: {
         position: 'relative',
-        minHeight: pxToRemWithUnit(48),
-        padding: `${pxToRemWithUnit(4)} ${pxToRemWithUnit(46)} ${pxToRemWithUnit(4)} ${pxToRemWithUnit(16)}`,
-        borderRadius: pxToRemWithUnit(4),
+        minHeight: '54px',
+        padding: '4px 54px 4px 12px',
+        borderRadius: borderRadiusSmall,
         border: 0,
         cursor: 'pointer',
         background: backgroundColor,
         color: primaryColor,
         textAlign: 'left',
         ...textSmallStyle,
         outline: 0,
-        ...getTagFocusJssStyle(primaryColor, hoverColor),
+        ...getTagFocusJssStyle(focusColor),
```

Comment:
We are passing colors due the fact that `getTagFocusJssStyle` should not call `getThemedColors` again (multiple calls). I improved it so that the themedColors object is passed and the function is only called once